tracer/test/Datadog.Trace.IntegrationTests/TelemetryTransportTests.cs

```diff
@@ -100,14 +97,26 @@ private static TelemetryData GetSampleData()
                 payload: null);
         }
 
+        private static ITelemetryTransport GetAgentOnlyTransport(Uri telemetryUri)
+        {
+            var transport = TelemetryTransportFactory.Create(
+                new TelemetrySettings(telemetryEnabled: true, configurationError: null, agentlessSettings: null, agentProxyEnabled: true),
+                new ImmutableExporterSettings(new ExporterSettings { AgentUri = telemetryUri }));
+            transport.Should().HaveCount(1);
+            transport[0].Should().BeOfType<AgentTelemetryTransport>();
+            return transport[0];
+        }
+
         internal class ErroringTelemetryAgent : MockTelemetryAgent<TelemetryData>
         {
             private readonly int _responseCode;
+            private readonly bool _isOutOfDateAgent;
 
-            public ErroringTelemetryAgent(int responseCode, int port)
+            public ErroringTelemetryAgent(int responseCode, int port, bool isOutOfDateAgent)
```

Comment:
Is `isOutOfDateAgent` set to true anywhere?

Reply:
Ah, probably not after I removed the agent version check in https://github.com/DataDog/dd-trace-dotnet/pull/2800/commits/8b7446d80b98e5aa53bfa02f543f0096feda0b91. Will fix, thanks!